docs/release-notes/8.4.asciidoc

```diff
@@ -0,0 +1,80 @@
+[[release-notes-header-8.4.0]]
+== 8.4
+
+[discrete]
+[[release-notes-8.4.0]]
+=== 8.4.0
+
+[discrete]
+[[known-issue-8.4.0]]
+==== Known issue
+* If additional look-back time is set for the advanced query rule preview, alerts from source documents that are outside the preview time frame may not appear in the preview ({pull}137422[#137422]).
+
+[discrete]
+[[breaking-changes-8.4.0]]
+==== Breaking changes
+// tag::breaking-changes[]
+// NOTE: The breaking-changes tagged regions are reused in the Elastic Installation and Upgrade Guide. The pull attribute is defined within this snippet so it properly resolves in the output.
+:pull: {pull}
+There are no breaking changes in 8.4.0.
+// end::breaking-changes[]
+
+[discrete]
+[[features-8.4.0]]
+==== Features
+* Creates a new rule type, New Terms, that creates an alert when a value appears for the first time in a particular field ({pull}134526[#134526]).
+* Adds the Insights section to the Alert details flyout to show related cases and alerts ({pull}136009[#136009], {pull}138419[#138419])
+* Shows process alerts in the event process analyzer ({pull}135340[#135340]).
+* Adds support for wildcard exceptions for detection rules. New operators are `matches` and `does not match` ({pull}136147[#136147]).
+* Adds a new search query parameter, `dry_run`, to the bulk actions API that allows you to simulate a bulk action without permanently updating rules ({pull}134664[#134664]).
+* Creates the response console, an interface that enables you to take actions on specific hosts ({pull}135360[#135360], {pull}134520[#134520]).
+* Enables a new method for the Task Manager API: `bulkUpdateSchedules`, which enables you to update the execution timing of `idle` tasks ({pull}132637[#132637]).
+* Includes integration policy errors and statuses in both {fleet} and {elastic-security} to help troubleshoot when an {agent} has an `Unhealthy` status.
+* Adds Attack surface reduction protections feature to reduce vulnerabilities on Windows endpoints. Credential hardening prevents attackers from stealing credentials stored in Windows system process memory.
+* Adds an endpoint self-healing feature to roll back file changes and processes on Windows endpoints when a prevention alert is generated by enabled protection features.
+* Adds the ability to run query packs as live queries ({pull}132198[#132198]).
+* Provides support for process, file, and network events in Kubernetes. You must enable the session view data setting on your {endpoint-cloud-sec} integration policy to enrich these events with session data and Kubernetes metadata fields.
+
+[discrete]
+[[bug-fixes-8.4.0]]
+==== Bug fixes and enhancements
+* Updates the Network page's UI to match the Hosts and Users pages ({pull}137541[#137541], {pull}136913[#136913]).
+* Fixes an error that could occur when you tried to apply an index pattern to rules using the bulk action option ({pull}134664[#134664]).
```

Comment:
It would be more accurate to say something like:
```
Improves bulk editing experience by informing users ahead of time of how many rules would be affected and unaffected by the action.
```

Reply:
Thanks for the review, @peluja1012! To clarify, did you want to remove the summary for 136418 and then add the PR link to the summary for 134664? It'd look like this:

Improves bulk editing experience by informing users ahead of time of how many rules would be affected and unaffected by the action (#134664), (136418).

Reply:
@nastasha-solomon strictly speaking, https://github.com/elastic/kibana/pull/136418 is about improving bulk export UX, and export is not bulk editing.

What do you think about this:

> Improves the experience of bulk editing index patterns by informing users ahead of time of how many machine learning rules would be unaffected by the action (https://github.com/elastic/kibana/pull/134664)

> Improves the experience of bulk exporting rules by informing users ahead of time of how many rules would be affected and unaffected by the action (https://github.com/elastic/kibana/pull/136418)

Reply:
Gotcha, thanks @banderror for pointing out that one's exporting and the other is for editing -- I was reading too fast and missed that detail. We can revise the summaries for both, thanks!